ui/raidboss/data/05-shb/raid/e12n.txt

```diff
@@ -6,5 +6,154 @@ hideall "--sync--"
+# is followed by:
+
+# Unknown_5879 > Cast > Cast > Malefecium
+# The time between 5861 Release and Unknown 5879
+# is consistent at 9 seconds for both Shiva and Titan.
+
+# It will then continue to the other of Titan/Shiva,
+# which will in turn be followed by the other closing pattern:
+
+# Stock > Formless Judgment > Unknown_5879 > Release
+# The time between Release and Stock is consistent at 14 seconds.
+
+# Shiva Block
+690.4 "Stock" sync /:Eden's Promise:5860:/
+700.0 "Junction Shiva" sync /:Eden's Promise:5862:/
+710.7 "Diamond Dust" sync /:Eden's Promise:5864:/ window 30,30
+716.8 "--sync--" sync /:Eden's Promise:5866:/
+721.9 "Frigid Stone" sync /:Eden's Promise:5867:/
+727.9 "Ice Floe" sync /:Eden's Promise:5868:/
+732.9 "Rapturous Reach" sync /:Eden's Promise:5878:/ window 30,30
+735.4 "Frigid Stone" # sync /:Eden's Promise:5869:/
+735.6 "--sync--" sync /:Eden's Promise:5871:/
+738.2 "--sync--" sync /:Eden's Promise:5879:/
+747.6 "Release" sync /:Eden's Promise:5861:/
+
+756.6 "--sync--" sync /:Eden's Promise:5879:/ jump 900.0
+761.5 "Stock?" sync /:Eden's Promise:5860:/ jump 1000.0
+766.2 "Cast?"
+770.2 "Formless Judgment?"
+776.9 "Cast?"
+782.4 "Release?"
+784.6 "Maleficium?"
+
+
+# Titan Block
+790.3 "Stock" sync /:Eden's Promise:5860:/
+800.0 "Junction Titan" sync /:Eden's Promise:5863:/
+810.7 "Earthen Fury" sync /:Eden's Promise:586A:/ window 30,30
+815.7 "Impact 1" sync /:Titanic Bomb Boulder:586E:/
+818.7 "Impact 2" # sync /:Bomb Boulder:586C:/
+822.7 "Explosion 1" sync /:Titanic Bomb Boulder:586F:/
+825.7 "Explosion 2" # sync /:Bomb Boulder:586D:/
+831.7 "Rapturous Reach" sync /:Eden's Promise:5878:/ window 30,30
+834.2 "Under The Weight" sync /:Eden's Promise:5870:/
+834.3 "--sync--" sync /:Eden's Promise:5871:/
+836.9 "--sync--" sync /:Eden's Promise:5879:/
+846.3 "Release" sync /:Eden's Promise:5861:/
+
+855.3 "--sync--" sync /:Eden's Promise:5879:/ jump 900.0
+860.2 "Stock?" sync /:Eden's Promise:5860:/ jump 1000.0
+864.9 "Cast?"
+868.9 "Formless Judgment?"
+875.6 "Cast?"
+881.1 "Release?"
+883.3 "Maleficium?"
+
+# Maleficium Block
+900.0 "--sync--" sync /:Eden's Promise:5879:/
+909.6 "Cast" sync /:Eden's Promise:4E2C:/
+920.3 "Cast" sync /:Eden's Promise:4E2C:/
+928.0 "Maleficium" sync /:Eden's Promise:5872:/ window 15,15
+
+940.9 "Stock" sync /:Eden's Promise:5860:/
+950.5 "Junction Shiva?" sync /:Eden's Promise:5862:/ window 15,15 jump 700.0
+950.5 "Junction Titan?" sync /:Eden's Promise:5863:/ window 15,15 jump 800.0
+961.2 "Diamond Dust?"
+961.2 "Earthen Fury?"
+966.2 "Impact 1?"
+969.2 "Impact 2?"
+972.4 "Frigid Stone?"
+973.2 "Explosion 1?"
+
+
+# Formless Judgment Block
+1000.0 "Stock" sync /:Eden's Promise:5860:/
+1008.7 "Formless Judgment" sync /:Eden's Promise:5873:/ window 15,15
+1011.5 "--sync--" sync /:Eden's Promise:5879:/
+1020.9 "Release" sync /:Eden's Promise:5861:/
+
+1034.8 "Stock" sync /:Eden's Promise:5860:/
+1044.4 "Junction Shiva?" sync /:Eden's Promise:5862:/ jump 700.0
```

Comment:
Is it random here which one happens next, or do they always alternate?

Reply:
They always alternate, but as I noted under the other comment, there's no way to know which, Shiva/Titan, follows Maleficium vs Judgment.